### PR TITLE
remove web3 (#34)

### PR DESCRIPTION
**Pull request by paulfreund94**

### CODE REVIEW

File: dApp/user/src/lib/Old/Eth.res [hunk 1]
```diff
@@ -1,67 +1,47 @@
...
 }
 @dead("+makeWithDefault")
 let makeWithDefault: (string, int) => t = (tokenId, default) =>
   switch make(tokenId) {
   | Some(wei) => wei
-  | None => default->Int.toString->BN.new_
+  | None => default ->BigNumber.fromInt
   }
-@dead("+makeFromInt") let makeFromInt: int => t = tokenId => tokenId->Int.toString->BN.new_
+@dead("+makeFromInt") let makeFromInt: int => t = BigNumber.fromInt
```
Comment (paulfreund94) [inline dApp/user/src/lib/Old/Eth.res:37]: not certain what the @dead annotation means... makeFromInt and makeFromEthStr have analogues which are built into Ethers so maybe we should remove them at some stage?

Comment (JasoonS) [reply dApp/user/src/lib/Old/Eth.res:37]: It does nothing, it's from here: https://github.com/rescript-association/reanalyze#dce-controlling-reports-with-annotations (dead code analysis)

It doesn't affect anything or change the behaviour of the code. It is just interesting to run that tool every now and again and see which code isn't being used

Comment (paulfreund94) [reply dApp/user/src/lib/Old/Eth.res:37]: Ah, thanks.

Comment (JasoonS) [reply dApp/user/src/lib/Old/Eth.res:37]: We can actually remove `Eth.res` too, sorry should have said that in the issue. (we should be able to do everything from this file with `Ethers.res`...)

Comment (paulfreund94) [reply dApp/user/src/lib/Old/Eth.res]: Done

**Review by JasoonS — APPROVED** [review]
Sweet :rocket:

**Pull request merged**